nodewatcher/modules/defaults/network_profile/defaults/wireless.py

```diff
@@ -152,17 +152,33 @@ def get_project_ssid(purpose, attribute='essid'):
 
             # Create mesh interface.
             if 'no-wifi-mesh' not in network_profiles:
-                self.setup_interface(
-                    state,
-                    cgm_models.WifiInterfaceConfig,
-                    parent=wifi_radio,
-                    configuration={
-                        'mode': 'mesh',
-                        'essid': get_project_ssid('mesh'),
-                        'bssid': get_project_ssid('mesh', attribute='bssid'),
-                        'routing_protocols': routing_protocols,
-                    },
-                )
+                if 'wifi-mesh-80211s' in network_profiles:
+                    # Configure 802.11s mesh interface.
+                        self.setup_interface(
```

Comment:
This indentation doesn't seem ok. BTW, I see that now both branches are almost the same, only the `mode` is different. Perhaps we could just unify both and have something like this instead:
```python
configuration={
    'mode': 'mesh11s' if 'wifi-mesh-80211s' in network_profiles else 'mesh',
    " ...
```